scripts/spack/packages/axom/package.py

```diff
@@ -242,6 +242,10 @@ def initconfig_hardware_entries(self):
             # CUDA_FLAGS
             cudaflags = "-restrict --expt-extended-lambda "
 
+            # Pass through any cxxflags to the host compiler via nvcc's Xcompiler flag
+            host_cxx_flags = spec.compiler_flags['cxxflags']
+            cudaflags += ' '.join(['-Xcompiler=%s ' % flag for flag in host_cxx_flags])
```

Comment:
This should be pushed up to Spack's CachedCMakePackage... I'll remember to do it next time I update the Axom package.